Ghidra 9.2 support

Ghidra 9.2 released and trying to compile this results in the following error:
```
ghidra-emotionengine/src/main/java/ghidra/emotionengine/MipsR5900AddressAnalyzer.java:335: error: cannot find symbol
                                                          instr.getMinAddress().getAddressSpace().getBaseSpaceID(),
                                                                                                 ^
  symbol:   method getBaseSpaceID()
  location: interface AddressSpace
```

--- a/src/main/java/ghidra/emotionengine/MipsR5900AddressAnalyzer.java
+++ b/src/main/java/ghidra/emotionengine/MipsR5900AddressAnalyzer.java
@@ -332,7 +332,7 @@ public class MipsR5900AddressAnalyzer extends ConstantPropagationAnalyzer {
 									}
 								}
 								symEval.makeReference(context, lastSetInstr, -1,
-									instr.getMinAddress().getAddressSpace().getBaseSpaceID(),
+									instr.getMinAddress().getAddressSpace().getSpaceID(),
 									unsignedValue, 1, RefType.DATA, PcodeOp.UNIMPLEMENTED, true,
 									monitor);
 								if (gp_assumption_value == null) {

